packages/kit/src/core/sync/write_client_manifest.js

```diff
@@ -47,11 +49,12 @@ export function write_client_manifest(kit, manifest_data, output, metadata) {
 			write_if_changed(`${output}/nodes/${i}.js`, generate_node(node));
 			return `() => import('./nodes/${i}')`;
 		})
+		.slice(0, client_routing ? manifest_data.nodes.length : 2)
```

Comment:
I _assume_ this line is creating a manifest with the root layout/error page even when server routing is enabled, but it might be good to add a comment explaining it